static/js/src/advantage/subscribe/react/components/Form/ProductType/ProductType.tsx

```diff
@@ -22,7 +22,7 @@ const PublicCloudInfo = {
       },
       {
         CTAName: "More info for Ubuntu 18.04 LTS users",
```

Comment:
Should the text for all 4 CTAs be updated as well?

```suggestion
        CTAName: "More info for Ubuntu 20.04 LTS users",
```